### ISSUE TITLE
Fast.h dependency EDIT: (Now optional)

### ISSUE THREAD

**Issue by fabio1955** [GitMoDu/FastX9CXXX#3]
Hi!
running your example (RandomValue) I got the following error:
...Documents\Arduino\libraries\FastX9CXXX-master/FastX9CXXX.h:13:18: fatal error: Fast.h: No such file or directory
Is it my fault?

**Comment by per1234** [comment]
Partially, though I think the lack of documentation is also to blame.

From https://github.com/GitMoDu/FastX9CXXX/blob/master/FastX9CXXX.h#L4:
```
///  Depends on Fast for IO https://github.com/GitMoDu/Fast
```
You need to install the Fast library: https://github.com/GitMoDu/Fast

**Comment by GitMoDu** [comment]
Yes, I don't have the patience to optimize direct port access for every different project, so I just use my small library. You can easily replace IO writing with digitalWrite, if you don't want the dependency.

**Comment by GitMoDu** [comment]
I've included the same information in the Readme in 72b2412d7a24265cbfa1de5267cfa57fe4b054c2. Thank you.

**Comment by GitMoDu** [comment]
The Fast library has been separated from the **_master_** branch, into the **_fast_** branch. So no need for the Fast library unless you want an optimized version for AVR only.
